lib/pom.xml

```diff
@@ -21,6 +21,10 @@
     </properties>
 
     <dependencies>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-web</artifactId>
```

Comment:
Do we really need this dependency for lib?

Reply:
Yes I add it cause some reasons